src/playground-file-system-controls.ts

```diff
@@ -22,11 +22,15 @@ import {
 } from 'lit-element';
 import {nothing} from 'lit-html';
 
-import '@material/mwc-menu/mwc-menu-surface.js';
+// Note despite usual best practices, we should _not_ import
+// @material/mwc-list-item directly, because @material/mwc-list already imports
+// it, and this causes a duplicate registration error on unpkg.com because of
+// redirects.
```

Comment:
even if everything resolve to the same version? I would have thought with ?module that the final URL is written into the import